lib/shape.mli

```diff
@@ -9,7 +9,7 @@ val line : ?x1:int -> ?y1:int -> int -> int -> shape
 val translate : int -> int -> shape -> shape
 val show : shape list -> unit
 val scale : float -> shape -> shape
-val rotate : int -> shape -> shape
+val rotate : float -> shape -> shape
```

Comment:
Why was this change for types made?

I think it's sufficient to support 1-360 degrees and not something in between.

Reply:
Alright, thank you